docs/administration/User_privilege.md

```diff
@@ -420,15 +420,15 @@ We recommend you customize roles to manage privileges and users. The following e
 1. Grant global read-only privilege:
 
    ```SQL
-   -- Create a role.
+   --Create a role.
    CREATE ROLE read_only;
-   -- Grant USAGE privileges on all catalogs to the role.
+   --Grant USAGE privileges on all catalogs to the role.
    GRANT USAGE ON ALL CATALOGS TO ROLE read_only;
-   -- Grant SELECT privileges on all tables to the role.
+   --Grant SELECT privileges on all tables to the role.
    GRANT SELECT ON ALL TABLES IN ALL DATABASES TO ROLE read_only;
-   -- Grant SELECT privileges on all views to the role.
+   --Grant SELECT privileges on all views to the role.
    GRANT SELECT ON ALL VIEWS IN ALL DATABASES TO ROLE read_only;
-   -- Grant SELECT privileges on all materialized views and the privilege to accelerate queries with them to the role.
+   --Grant SELECT privileges on all materialized views and the privilege to accelerate queries with them to the role.
```

Comment:
Grant SELECT privileges on all materialized views to the role to accelerate queries?